Source/ScreenShotUtils.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "Config.h"
+#include "PS2VM.h"
+
+class CScreenShotUtils : public Framework::CConfig
+{
+public:
+    typedef std::function<void(int, const char*)>        Callback;
+
+    static void                             TriggerGetSceenshot(CPS2VM*, Callback);
+
+private:
+    static CConfig::PathType                GetScreenShotDirectoryPath();
+    static CConfig::PathType                GetGameScreenShotDirectoryPath(std::string gameID);
```

Comment:
It's mostly for cosmetics and from organisational point of view, without folders can get messy really quick, (I know this isn't PS4, but on my PS4, I'd take atleast 100 screenshot per game, I'd imagine that'd be alot of images to scroll through), so I'll leave this choice to you.